src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1301,9 +1304,22 @@ export class AmpStoryPlayer {
   rewind(storyUrl) {
     const story = this.getStoryFromUrl_(storyUrl);
 
-    story.messagingPromise.then((messaging) =>
-      messaging.sendRequest('rewind', {})
-    );
+    this.whenConnected_(story)
+      .then(() => story.messagingPromise)
```

Comment:
Just curious, what happens if we remove the `whenConnected` and just wait for the messagingPromise to be available instead?

Reply:
`messagingPromise` would be undefined when the story isn't connected yet, since it's only initialized on `appendToDom_()`.